src/Tests/Microsoft.Diagnostics.Monitoring.TestCommon/TestAppScenarios.cs

```diff
@@ -49,6 +49,7 @@ public static class Exceptions
             public static class SubScenarios
             {
                 public const string SingleException = nameof(SingleException);
+                public const string FilteringExceptions = nameof(FilteringExceptions);
```

Comment:
nit: There isn't anything in this scenario that relates to filtering. It should be renamed to something like `MultipleExceptions`